CMakeLists.txt

```diff
@@ -2608,7 +2608,14 @@ if(HID)
   find_package(hidapi)
   # hidapi_VERSION is only available starting with 0.10.0
   if(NOT hidapi_FOUND OR NOT hidapi_VERSION OR hidapi_VERSION VERSION_LESS 0.10.1)
-    message(STATUS "Linking internal libhidapi statically")
+    if(hidapi_FOUND)
+        if(hidapi_VERSION)
+            message(STATUS "Found libhidapi version: ${hidapi_VERSION}")
+        else()
+            message(STATUS "Found libhidapi version < 0.10.0")
+        endif()
+    endif()
+    message(STATUS "Linking internal libhidapi 0.10.1 statically")
```

Comment:
This is error prone. I do not anticipate that anyone would remember to update this line when updating the vendored library.